src/transport/raw/TCP.h

```diff
@@ -104,15 +104,7 @@ class DLL_EXPORT TCPBase : public Base
             size_t packetsBuffersSize) :
         mActiveConnections(activeConnectionsBuffer),
         mActiveConnectionsSize(bufferSize), mPendingPackets(packetBuffers), mPendingPacketsSize(packetsBuffersSize)
-    {
-        PendingPacket emptyPending{
-            .peerAddress  = PeerAddress::Uninitialized(),
-            .packetBuffer = nullptr,
-        };
-
-        std::fill(mActiveConnections, mActiveConnections + mActiveConnectionsSize, nullptr);
```

Comment:
Why do we no longer need to fill `mActiveConnections` with null?  This part doesn't seem right to me....